docker/cp2k/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:18.04
+
+# Install ssh, which is a runtime dep of the ubuntu cp2k package
+# (due to its usage of MPI)
+RUN apt-get update && \
+  apt-get install -y ssh git
```

Comment:
I don't understand why this isn't just one single RUN command, the package manager should only get updated once, and it will figure out the dependencies with a single install line won't it?

Reply:
I think I was doing them one at a time to speed up development. But then I never went back and merged them all for the final Dockerfile. I can merge them together.

Reply:
This is fixed.